spec/factories/generic_event.rb

```diff
@@ -22,6 +22,17 @@
     end
   end
 
+  factory :per_incident, parent: :generic_event do
```

Comment:
Not sure this really matters but looking at the file this is getting very large, admittedly is a Factory.

Reply:
Yeah, I reckon it is worth splitting this file out for sure.